app/modules/transactions.js

```diff
@@ -77,14 +77,20 @@ const makeRequest = (sendEntries: Array<SendEntryType>, config: Object) => {
     config.tokensBalanceMap
   )
 
+  const priorityFees = sendEntries.reduce((accum, currentValue) => {
+    const priorityFeeAsNumber = Number(currentValue.priorityFee)
+    const total = priorityFeeAsNumber + accum
+    return total
+  }, 0)
+
   if (script === '') {
     return api.sendAsset({ ...config, intents: buildIntents(sendEntries) }, api.neoscan)
   } else {
     return api.doInvoke({
       ...config,
       intents: buildIntents(sendEntries),
       script,
-      gas: 0
+      gas: priorityFees
```

Comment:
@snowypowers if you could take a look specifically here (Im pretty sure this is incorrect as it is not working currently) and explain where I need to attach priority fees this PR will be 👍

Reply:
```

Reply:
Thanks!